StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetLoader.swift

```diff
@@ -316,9 +318,25 @@ final class PaymentSheetLoader {
 
         // Move default PM to front
         if let customerID = configuration.customer?.id {
-            let defaultPaymentMethod = CustomerPaymentOption.defaultPaymentMethod(for: customerID)
+            var defaultPaymentMethodOption: CustomerPaymentOption?
+            // read from back end
+            if configuration.allowsSetAsDefaultPM,
+               let customer =  elementsSession.customer {
```

Comment:
Similar here w/ the comment and nested if's and multiple branches leading to the same `CustomerPaymentOption.defaultPaymentMethod(for: customerID)`. I think there's a cleaner way to write this.